app/assets/stylesheets/custom.scss

```
@@ -0,0 +1,35 @@
+@import 'bootstrap-sprockets';
+@import 'bootstrap';
+
+/* universal */
+body {
+padding-top: 60px;
```

indent sai em nhé (thụt lề)
```suggestion
  padding-top: 60px;
```

self check file này